data/presets/leisure/bathing_place.json

```diff
@@ -0,0 +1,45 @@
+{
+    "aliases": [
+        "Bathing spot",
+        "Dip spot",
+        "Dipping spot",
+        "Swim spot",
+        "Swimming spot",
+        "Wild swimming spot"
+    ],
+    "icon": "fas-water-ladder",
+    "fields": [
+        "name",
+        "access_simple",
+        "shower",
+        "toilets",
+        "drinking_water_available",
+        "picnic_table"
+    ],
+    "moreFields": [
+        "opening_hours",
+        "operator",
+        "fee",
+        "payment_multi_fee",
+        "charge_fee",
+        "supervised",
+        "lifeguard_check",
```

Comment:
Is this tag used for kind of official places as well? Or do they become something else once they are paid and supervised?

Reply:
That would normally make it an `amenity=public_bath` or possibly a `leisure=beach_resort`. So although a few dozen `bathing_place`s have `fee`, `opening_hours`, `supervised` and/or `lifeguard`, the answer is almost always `no` and `24/7`. That's why I put it in `moreFields`, not in `fields`. But I am not very familiar with the conventions of id-tagging-schema - let me know if this means that, for example, it should not be in the preset at all.

Reply:
I think about it this way…
- add fields/moreFields that are already used a lot in the data
- add fields/moreFields that are very helpful to have and documented/"in use"
- don't add fields/moreFields that we don't want new Users to add

So in this case the question seems to be if we want to give visibility to data that is present but maybe wrong because it should be a different primary tag. I think we should not add those fields. Experienced users will see the data regardless. But less experienced users will not add "false" data. Experienced users can change the preset and then the data will be visible.